setup.py

```diff
@@ -1,23 +1,13 @@
-import os
-
 from setuptools import setup
 
-
-def read_version(package):
-    with open(os.path.join(package, "__init__.py"), "r") as fd:
-        for line in fd:
-            if line.startswith("__version__ = "):
-                return line.split()[-1].strip().strip("'")
-
-
-VERSION = read_version("iolite")
+from iolite import __version__
 
 setup(
-    description="",
+    description="API client for interacting with IOLite's remote API",
```

Comment:
@firstdayofjune do we even need setup.py? 

Since when we will eventually distribute this it'll be packaged up via `poetry build` and published to pypi

p.s. feel free to add yourself to the authors section on pyproject.toml

Reply:
I guess this actually is the right question to ask here.

I never published anything to PyPI, so I read about how to do it using poetry [here](https://www.brainsorting.com/posts/publish-a-package-on-pypi-using-poetry/) (which btw. is pretty straight-forward with a`publish` command available in [poetry](https://python-poetry.org/docs/cli/#publish)).
No mention of a `setup.py` file.
Also, after reading the [PEP-518](https://www.python.org/dev/peps/pep-0518/), I understand the `pyproject.toml`'s whole  purpose is to replace `setup.py` and the like.

So I agree to remove the `setup.py` completely 👍

Reply:
Makes total sense 👍🏼 

Yeah publishing should be super straight forward with poetry!